sdk/dynamicconfig/example/server/server.go

```diff
@@ -37,12 +37,26 @@ type server struct {
 // GetConfig implemented DynamicConfigServer
 func (s *server) GetConfig(ctx context.Context, in *pb.ConfigRequest) (*pb.ConfigResponse, error) {
 	log.Printf("Config being read\n")
-	schedule := pb.ConfigResponse_MetricConfig_Schedule{Period: 1}
+
+	pattern1 := pb.ConfigResponse_MetricConfig_Schedule_Pattern{
+		Match: &pb.ConfigResponse_MetricConfig_Schedule_Pattern_StartsWith{ StartsWith: "One" },
+	}
+	schedule1 := pb.ConfigResponse_MetricConfig_Schedule{
+		InclusionPatterns: []*pb.ConfigResponse_MetricConfig_Schedule_Pattern{&pattern1},
+		Period: 1,
+	}
+	pattern2 := pb.ConfigResponse_MetricConfig_Schedule_Pattern{
+		Match: &pb.ConfigResponse_MetricConfig_Schedule_Pattern_Equals{ Equals: "Two Metric" },
+	}
+	schedule2 := pb.ConfigResponse_MetricConfig_Schedule{
+		InclusionPatterns: []*pb.ConfigResponse_MetricConfig_Schedule_Pattern{&pattern2},
+		Period: 5,
```

Comment:
What do you think about using the enum values, rather than a literal, to reinforce the idea that there should only be a set number of periods possible? So in this case, it might look like `pb.ConfigResponse_MetricConfig_Schedule_SEC_5`. Much more verbose, but potentially safer behavior?

Reply:
Good idea! I'd like to wait to see if the maintainers rather we just have an int in the protocol.